### ISSUE TITLE
An error occurs when building flutter for Android

### ISSUE THREAD

**Issue by sgbai78** [phosphor-icons/flutter#19]
An error occurs when building flutter for Android.

I saw the warning message and added the '--no-tree-shake-icons' option and the build succeeded, but I think this should be a patch.

cmd:
flutter build apk

error:
This application cannot tree shake icons fonts. It has non-constant instances of IconData at the following locations:
  - file:///C:/Users/sgbai/AppData/Local/Pub/Cache/hosted/pub.dev/phosphor_flutter-2.0.0/lib/src/phosphor_icons_bold.dart:8:23
  - file:///C:/Users/sgbai/AppData/Local/Pub/Cache/hosted/pub.dev/phosphor_flutter-2.0.0/lib/src/phosphor_icons_bold.dart:11:29

...

Target aot_android_asset_bundle failed: Exception: Avoid non-constant invocations of IconData or try to build again with --no-tree-shake-icons.

**Comment by CodeDoctorDE** [comment]
Duplicate of #16

**Comment by ugurcany** [comment]
any eta for the release of this fix?

**Comment by CodeDoctorDE** [comment]
I'm not the maintainer of this package, I'm just the one that fixed this bug.
For now you can use this:
```
dependencies:
  phosphor_flutter:
    git:
      url: https://github.com/phosphor-icons/flutter
      ref: f370dd2c25d3ea51ffb8f1c7c183bc9f33889c82
```

**Comment by rektdeckard** [comment]
Resolved in #18
